version.mak

```diff
@@ -1,8 +1,8 @@
 # Version Information
 
 VERSION_MAJOR=1
-VERSION_MINOR=4
-VERSION_PATCH_LEVEL=5
+VERSION_MINOR=5
+VERSION_PATCH_LEVEL=0
```

Comment:
Hmm, isn't it  above bumping from `1.4.5` to `1.5.0`? If so, please fix the PR/commit title

Reply:
Yes sorry I meant ovirt 4.5.0, metrics will be 1.5.0.
see https://github.com/oVirt/ovirt-engine-metrics/commit/aabf26afeabf9a51965ec6fe85c7669ddd12277c